### PR TITLE
fix tls warnings

### PR DESCRIPTION
**Pull request by UplinkCoder**
@RazvanN7 Why did you change this in the first place ?

### CODE REVIEW

File: src/ddmd/astnull.d [hunk 1]
```diff
@@ -38,47 +38,47 @@ struct ASTNull
     alias PROTexport        = PROTKIND.PROTexport;
     alias PROTundefined     = PROTKIND.PROTundefined;
 
-    static int MODconst            = 0;
```
Comment (mathias-lang-sociomantic) [inline src/ddmd/astnull.d]: Make it `static immutable int` and you get the best of both worlds. It's an lvalue which is known at CT.

Comment (UplinkCoder) [inline src/ddmd/astnull.d]: this is not meant to be used.
it's just a template...
so I took the four letter word over 3 long words

**Comment by UplinkCoder** [comment]
@andralex merge please :)
project-tester failures are unrelated.

**Comment by andralex** [comment]
@UplinkCoder thanks for the ping!

**Review by andralex — APPROVED** [review]

**Comment by andralex** [comment]
cc @MartinNowak for conformity

**Pull request merged**